build.xml

```diff
@@ -87,6 +91,36 @@
         </artifact:mvn>
     </target>
 
+    <target name="compile-tests" depends="build" description="Compile test files">
+        <path id="test.classpath">
+            <path refid="project.class.path" />
+            <fileset dir="lib" includes="**/*.jar" />
+        </path>
+        <javac srcdir="${test.src}"
+               destdir="${build}"
+               classpathref="test.classpath"
+               includeantruntime="false">
+        </javac>
+    </target>
+
+    <target name="test" depends="compile-tests" description="Runs all Robolectric unit tests.">
+        <path id="junit.classpath">
+            <path refid="project.class.path" />
```

Comment:
It seems the junit classes are inside the robolectric jar, no need of a external junit jar.